### PR TITLE
Fix card behavior with long title

### LINKED ISSUE #8: Text overflow on card
The text overflows when the card title is too long

capture d ecran 2017-01-02 a 12 17 43

### PR DESCRIPTION
**Pull request by jrmneveu**
Display the entire title in the card and avoid the title to go outside the card.
fixes #8

### CODE REVIEW

**Comment by asdine** [comment]
Still doesn't work when there is only one word:

capture d ecran 2017-01-08 a 15 46 41

File: desktop/app/scss/pulpe-structure/pulpe-cards.scss [hunk 1]
```diff
@@ -6,6 +6,8 @@
   h3 {
     font-size: .875rem;
     font-weight: 400;
+    white-space: normal;
+    word-wrap:break-word
```
Comment (asdine) [inline desktop/app/scss/pulpe-structure/pulpe-cards.scss]: You should use atom-beautifier and active the Scss support

Comment (jrmneveu) [inline desktop/app/scss/pulpe-structure/pulpe-cards.scss]: Done, thanks

Comment (asdine) [inline desktop/app/scss/pulpe-structure/pulpe-cards.scss]: Use the `Pretty diff` mode + `Align assignments`

Comment (jrmneveu) [inline desktop/app/scss/pulpe-structure/pulpe-cards.scss]: Done, thanks

**Pull request merged**